presto-kudu/bin/run_kudu_tests.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#
+# Runs Kudu tests with single tablet server by default.
+# If '3' is given as argument, it runs tests with three tablet servers (and num_replicas = 3).
+
+set -euo pipefail -x
+
+export KUDU_MASTER_RPC_PORT=17051
+export DOCKER_HOST_IP=$(ip -4 addr show docker0 | grep -Po 'inet \K[\d.]+')
```

Comment:
Hoping new variant with `docker network inspect bridge ...` is compatible with MacOS.

Reply:
Why not use port mapping when running docker container, and use `localhost:<theport>` for `-Dkudu.client.master-addresses=` ?

Reply:
The problem is, that the Kudu master advertises the addresses of the tablet servers. I.e. if a client creates a scanner for a Kudu table, it asks the Kudu master for the partitions which need to be contacted for retrieving the data. The response contains the advertised addresses of the tablet servers which can deliver these partitions. Therefore the Kudu master must know the external IP addresses of the tablet servers, i.e the IP address, which can be reached by the client.

Reply:
Sounds reasonable. So you're basically mimicking `presto-hive-hadoop2/bin/run_hive_tests.sh`, but your connector probably doesn't support SOCKS proxy, so this doesn't "just work".
Did you consider following product tests's path? I know it won't be super-trivial, but if successful, this would be more portable:
- build test jar (uber, with all the deps, or a bunch of jars)
- using docker compose start kudu container(s) and a container for tests
- now the test container "sees" other containers just as nodes on a network (thanks to docker network)

@kokosing what do you think?

Reply:
I was thinking about that. There couple of things which makes me prefer to leave it as it is:
 - there are not much generic tests in product tests, so new tests would need be created to just to test this connector. Now this connector is using the same test suites as other connectors.
 - it is a bit easier to debug connector with these tests, also it is easier to write more low level tests than product tests. For example unit tests for `KuduMetadata`, while in product tests you talk to Presto via JDBC.

Thing that might hit us in future is this socks proxy, but it should not be hard to add in if needed.

Another idea would be to run integration tests in docker container, then proxy would not be needed. This has also its own pros and cons.